includes/alerts.inc.php

```diff
@@ -37,7 +37,7 @@ function GenSQL($rule) {
         return false;
     }
     //Pretty-print rule to dissect easier
-    $pretty = array('*'  => '*', '('  => ' ( ', ')'  => ' ) ', '/'  => '/', '&&' => ' && ', '||' => ' || ', 'DATE_SUB ( NOW (  )' => 'DATE_SUB( NOW()');
+    $pretty = array('*'  => '*', '('  => ' ( ', ')'  => ' ) ', '/'  => '/ ', '&&' => ' && ', '||' => ' || ', 'DATE_SUB ( NOW (  )' => 'DATE_SUB( NOW()');
```

Comment:
It's fine to put it in this one

On 24 May 2016 18:44:25 CEST, David Bell <notifications@github.com> wrote:
>> @@ -37,7 +37,7 @@ function GenSQL($rule) {
>>          return false;
>>      }
>>      //Pretty-print rule to dissect easier
>> -    $pretty = array('*'  => '*', '('  => ' ( ', ')'  => ' ) ', '/' 
>=> '/', '&&' => ' && ', '||' => ' || ', 'DATE_SUB ( NOW (  )' =>
>'DATE_SUB( NOW()');
>> +    $pretty = array('*'  => '*', '('  => ' ( ', ')'  => ' ) ', '/' 
>=> '/ ', '&&' => ' && ', '||' => ' || ', 'DATE_SUB ( NOW (  )' =>
>'DATE_SUB( NOW()');
>
>Yeah, accidentally branched off my old PR :( Want me to submit fresh,
>or are you OK with my revert in the next commit?
>
>---
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/librenms/librenms/pull/3548/files/45b331b1008d99ca2683b5ed27436f77d41c0287#r64429332